README.md

```diff
@@ -16,33 +16,16 @@
   </a>
 </p>
 
-![Hero Image with Logo](./.github/hero2.png)
+![Hero Image with Logo](https://raw.githubusercontent.com/microsoft/react-native-windows/master/.github/hero2.png)
```

Comment:
is this change needed? thinking about checking if a link exists, it's cheaper to check a file that is local to the CI loop than to hit the network (plus github can throttle us if we hit too many urls in a short period of time)